test/unit/marked-spec.js

```diff
@@ -25,3 +25,11 @@ describe('Test paragraph token type', function () {
     expect(tokens[7].type).toBe('text');
   });
 });
+
+var inputAutolinkBold = '__test@test.com__';
+var expectedOutputAutolinkBold = '<p><strong><a href="mailto:test@test.com">test@test.com</a></strong></p>\n';
```

Comment:
Can you put these 2 variables inside the test so it matches the other ones?

Reply:
This test should be added to [/test/specs/marked/marked.json](https://github.com/markedjs/marked/blob/master/test/specs/marked/marked.json) instead of creating a unit test

Reply:
Will do right now, didn't find that one sorry.

Reply:
Done 👍